pkg/suggestion/v1alpha3/internal/search_space.py

```diff
@@ -50,12 +50,12 @@ def convertParameter(p):
 
 
 class HyperParameter(object):
-    def __init__(self, name, type, min, max, list, step):
+    def __init__(self, name, type_, min_, max_, list_, step):
```

Comment:
In Python, These are common practice to avoid shadows of built-in methods and variables.

Reply:
LGTM